arber-salihu/loGIStics
# issue-9: Initilize gradle project with spring boot

== Summary of Changes ==
I added a new spring boot project to the base of the repo, under the snow-api directory. This project is set up with gradle - groovy, and will produce a war.

The added dependencies include:
 - Spring Boot DevTools
 - PostgreSQL Driver
 - Spring Web
 - Lombok

== Test Steps ==
1. Checkout the `issue-9` branch
2. cd into the snow-api directory
3. Run `./gradlew clean build` to build the backend
4. Verify it builds successfully
